src/Catalyst.Node.POA.CE/Program.cs

```diff
@@ -98,6 +102,21 @@ private static async Task CustomBootLogic(Kernel kernel)
             RegisterNodeDependencies(Kernel.ContainerBuilder);
 
             kernel.StartContainer();
+
+            //var registrations = kernel.Instance.ComponentRegistry.Registrations;
+
+            //var providerRegistrations = registrations.Where(r => r.Activator.LimitType == typeof(StateProvider)).ToArray();
+            //var stateProviders = providerRegistrations
+            //   .Select(r => kernel.Instance.ResolveComponent(r, new Parameter[0]))
+            //   .ToArray();
+
+            //var executorRegistrations = registrations.Where(r => r.Activator.LimitType == typeof(DeltaExecutor)).ToArray();
+            //var executors = executorRegistrations
+            //   .Select(r => kernel.Instance.ResolveComponent(r, new Parameter[0]))
+            //   .ToArray();
+
+            var executor = kernel.Instance.Resolve<IDeltaBuilder>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'executor'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)